tracer/build_in_docker.sh

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env bash
 set -euox pipefail
 
-# in case we are being run from outside this directory
```

Comment:
The only way I found to get build_in_docker.sh to works was to remove this line and run it from the source tree root.

Reply:
Please don't remove this, the correct fix is shown in https://github.com/DataDog/dd-trace-dotnet/commit/943dc89623c825dfd6dc0dee1609c486cdba9143

This was broken by the tracer directory move

Reply:
reverted in e39dbcff05c2af9cd990914568938d6793651134